Chiazokam/Politico
# #163498874 create the get results endpoint

#### What does this PR do?
This PR tests and creates the get results Endpont

#### Description of Task to be completed?
- implement feedback from LFA
- create the get results endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-get-results-163498874`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `GET` request to `http://localhost:3000/api/v1/office/:id/result`
- You should receive a successful response

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#163498874](https://www.pivotaltracker.com/story/show/163498874)

### Images
![image](https://user-images.githubusercontent.com/26940294/52067622-4330fd80-257b-11e9-94cb-0757a10f7740.png)
